account_operating_unit/report/account_invoice_report.py

```diff
@@ -19,10 +19,3 @@ def _select(self):
             ,line.operating_unit_id
         """
         return select_str
-
-    def _group_by(self):
```

Comment:
why is it just removed?

Reply:
It was removed from OC in v16 (https://github.com/odoo/odoo/pull/96134) and it seems that it's not used anymore outside of reports like account.invoice.report